projects/addon-doc/tokens/doc-icons.ts

```diff
@@ -13,11 +13,19 @@ export interface TuiDocIcons {
     readonly check: string;
     readonly link: string;
     readonly languages: string;
+    /**
+     * TODO: drop optional in v5
+     */
+    readonly externalLink?: string;
     /**
      * TODO: drop optional in v5
      */
     readonly shrink?: string;
     readonly expand?: string;
+    readonly ltr: {
```

Comment:
Same here. Please be smarter about such things, it is exactly the same thing in exactly the same place, we shouldn't have to point out every line on reviews.

Reply:
<img width="190" height="266" alt="image" src="https://github.com/user-attachments/assets/a6f7d1fe-1337-463d-9c0f-7e492ea1198b" />